db/setup/schema.sql

```diff
@@ -0,0 +1,41 @@
+DROP TABLE IF EXISTS rentals;
+DROP TABLE IF EXISTS movies;
+DROP TABLE IF EXISTS customers;
+CREATE TABLE movies(
+  id serial PRIMARY KEY,
+  title text,
+  overview text,
+  release_date date,
+  inventory integer
+);
+
+CREATE INDEX movies_title ON movies (title);
+CREATE INDEX movies_release_date ON movies (release_date);
+
+/*customers */
+CREATE TABLE customers(
+  id serial PRIMARY KEY,
+  name text,
+  registered_at text,
+  address text,
+  city text,
+  state text,
+  postal_code integer,
+  phone text,
+  account_credit decimal
```

Comment:
I'm guessing that this is for money. I wonder if it would better to use an integer and count in pennies? That is common practice.